domain/domain.go

```diff
@@ -605,6 +605,9 @@ func (do *Domain) isClose() bool {
 	return false
 }
 
+// DomainMapDelete is used to avoid cycle import with `session` package.
+var DomainMapDelete func (kv.Storage)
```

Comment:
How about adding a `callback' to Domain instead of another global variable?

Reply:
how can we pass the callback? domain package can't access session package

Reply:
like this
```go
type Domain struct {
	callback func()
}
```

Reply:
On the other hand,  `domain` is created by `domap.Get()`, so is `domap's` responsibility to call `Close`?